lib/DataHarmonizer.js

```diff
@@ -1762,6 +1762,96 @@ class DataHarmonizer {
     return mappedCell.join(delimiter);
   }
 
+  /**
+   * Given a table row, output a value based on the following conditional:
+   * ```
+   * if (value in headerNameToCheck == valToMatch) {
+   *   return value in headerNameToOutput;
+   * } else {
+   *   return "";
+   * }
+   * ```
+   * TODO is there any need for additional complexities in getMappedField?
```

Comment:
@ddooley I am just copy-pasting vals for the most part (with the exception of some trimming I also need advice on below)